spectral-ruleset-govuk-public/ruleset.yaml

```
@@ -0,0 +1,2 @@
+extends:
+  - [spectral:oas, all]
```

Did we want this?

```suggestion
  - - spectral:oas
    - all
```

(Unlike the usual request of reducing diff noise, this array is only going to contain two items)

---

I think stick with your original decision to leave as is if it's not going to change?

---

Sounds like a plan :+1: